llvm/lib/Target/AMDGPU/VOPCInstructions.td

```diff
@@ -1508,13 +1532,25 @@ multiclass VOPC_Real_with_name<GFXGen Gen, bits<9> op, string OpName,
         // the destination-less 32bit forms add it to the asmString here.
         VOPC_Real<ps32, Gen.Subtarget, asm_name#"_e32">,
         VOPCe<op{7-0}>;
-      def _e64#Gen.Suffix :
-            VOP3_Real_Gen<ps64, Gen, asm_name>,
-            VOP3a_gfx11_gfx12<{0, op}, ps64.Pfl> {
-        // Encoding used for VOPC instructions encoded as VOP3 differs from
-        // VOP3e by destination name (sdst) as VOPC doesn't have vector dst.
-        bits<8> sdst;
-        let Inst{7-0} = sdst;
+      if ps64.Pfl.IsRealTrue16 then {
+        def _e64#Gen.Suffix :
+           VOP3_Real_Gen<ps64, Gen, asm_name>,
+           VOP3e_t16_gfx11_gfx12<{0, op}, ps64.Pfl> {
+           // Encoding used for VOPC instructions encoded as VOP3 differs from
```

Comment:
This should be indented more. +Same below.

Reply:
done